.github/workflows/build-arch.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow builds the AYAB package for Arch Linux
+# and stores it as an artifact
+
+name: build-arch
+
+on:
+  push:
+    branches:
+      - master
+      # - '*-dev'
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        python-version: [3.6]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Set GIT_TAG variable
+      run: echo "::set-env name=GIT_TAG::$(git describe --tags)"
+    - name: Set PACKAGE_VERSION
+      run: |
+        echo ${{env.GIT_TAG}} > src/main/resources/base/ayab/package_version
+        sed -i 's/PACKAGE_VERSION/${{env.GIT_TAG}}/' src/build/settings/base.json
+        cat src/main/resources/base/ayab/package_version
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Use old pip version
+      run: python3 -m pip install pip==18.1
+    - name: Update setuptools
+      run: python3 -m pip install --upgrade setuptools
+    - name: Install fbs
+      run: python3 -m pip install fbs==0.8.6
+    - name: Create Docker image
+      run: |
+        echo '{"gpg_pass": "password"}' > src/build/settings/secret.json
```

Comment:
I can store the gpg password as secred in the github repo (see https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository), please send it to me over a secure channel.

Reply:
This is a dummy password anyway! We can do this when the scripts are more mature.